app/src/pages/inside/logsPage/logsPage.jsx

```diff
@@ -31,40 +30,33 @@ import {
   WITH_ATTACHMENTS_FILTER_KEY,
   HIDE_PASSED_LOGS,
   HIDE_EMPTY_STEPS,
-  getLogLevel,
   setLogLevel,
   DETAILED_LOG_VIEW,
   logViewModeSelector,
   LOG_STATUS_FILTER_KEY,
-  isLogPageWithNestedSteps,
 } from 'controllers/log';
 import { parentItemSelector } from 'controllers/testItem';
 import { withFilter } from 'controllers/filter';
 import { debugModeSelector } from 'controllers/launch';
-import { withPagination, PAGE_KEY, DEFAULT_PAGINATION, SIZE_KEY } from 'controllers/pagination';
+import { withPagination, PAGE_KEY } from 'controllers/pagination';
 import { withSortingURL, SORTING_ASC } from 'controllers/sorting';
 import { userIdSelector } from 'controllers/user';
-import { PaginationToolbar } from 'components/main/paginationToolbar';
 import { LOG_PAGE, LOG_PAGE_EVENTS } from 'components/main/analytics/events';
 import { SpinningPreloader } from 'components/preloaders/spinningPreloader';
 import { TestItemLogsToolbar } from './testItemLogsToolbar';
 import { LogToolbar } from './logToolbar';
 import { HistoryLine } from './historyLine';
 import { LogItemInfo } from './logItemInfo';
-import { LogsGrid } from './logsGrid/logsGrid';
-import { LogsGridToolbar } from './logsGridToolbar';
-import { SauceLabsSection } from './sauceLabsSection';
+import { LogsGridWrapper } from './logsGridWrapper';
 
 @connect(
   (state) => ({
-    logItems: logItemsSelector(state),
     loading: loadingSelector(state),
     pageLoading: pageLoadingSelector(state),
     userId: userIdSelector(state),
     debugMode: debugModeSelector(state),
     logViewMode: logViewModeSelector(state),
     parentItem: parentItemSelector(state),
-    isNestedStepView: isLogPageWithNestedSteps(state),
   }),
   {
     refresh: refreshLogPageData,
```

Comment:
Inside of this component you should delete all unused methods, variables, and so on (f.i. `onChangeHideEmptySteps`). Double-check this.

Reply:
I don't understand why a @connectRouter is needed and how it works, I can delete something unnecessary